superset-frontend/cypress-base/cypress/integration/dashboard/filter.js

```diff
@@ -56,8 +56,26 @@ export default () =>
       cy.get('.Select__control')
         .contains('Select [region]')
         .click({ force: true });
+
+      // should open the filter indicator
+      cy.get('.filter-indicator.active')
+        .should('be.visible')
+        .should($node => {
```

Comment:
why prefix node with a dollar sign? Can we get rid of that in the variable name?

Reply:
Was just following Cypress's official doc, but I agree it's probably not necessary.